### DISCUSSION TITLE
New version of diamond template with CSS layer support

### DISCUSSION THREAD

**Discussion by mklinke** [primefaces/community#612] [PrimeNG Templates] [answered]
It's been a couple of weeks since we attempted to upgrade to PrimeNG 16. However, we weren't able to finish the upgrade due to issues with the new [CSS layer implementation](https://github.com/primefaces/primeng/issues/13737) in PrimeNG 16.4 and the latest Diamond template v16.0.0 (from Sep 22, 2023). Apparently, this template doesn't support the CSS layer change yet and it messes up our application styles. See also [this prior discussion](https://github.com/orgs/primefaces/discussions/269).

We can't go to a lower 16.X due to [another bug](https://github.com/primefaces/primeng/issues/12945) that we're experiencing w/ Jest.

Is there a plan or at least some way to tell when (and I hope I don't have to ask if) there will be an update to the Diamond template (a premium template that we payed for) with CSS layer support?

Thanks in advance,
Martin

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

We're planning to upgrade Diamond to v17 soon. With the update, we'll address this issue. \
Thanks for reporting!
